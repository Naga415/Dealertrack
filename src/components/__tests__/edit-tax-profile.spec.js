import React from "react";
import renderer from "react-test-renderer";
import EditTaxProfile from "../edit-tax-profile";

describe("EditTaxProfile", () => {
  it("renders <EditTaxProfile />", () => {
    const props = {
      htmlId: "EditTaxProfile",
      name: "Edit Taxes",
      noTaxes: false,
      state: "NY",
      zipcode: "11042",
      isDisabled: false,
      actions: {}
    };

    const component = renderer.create(<EditTaxProfile {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
