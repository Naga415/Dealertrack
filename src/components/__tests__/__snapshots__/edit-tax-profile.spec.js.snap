// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditTaxProfile renders <EditTaxProfile /> 1`] = `
<div
  className="row"
  id="edit-tax-profile"
>
  <div
    className="col-xs-offset-half col-md-3 col-xs-4"
    id="noTaxes"
  >
    <div
      className="checkbox"
    >
      <label
        htmlFor="noTaxes"
      >
        <input
          checked={false}
          disabled={false}
          name="noTaxes"
          onChange={[Function]}
          type="checkbox"
        />
        No Taxes
      </label>
    </div>
  </div>
  <div
    className="col-xs-5"
    id="taxesProfile"
  >
    <div
      className="col-xs-12 form-group-taxes"
    >
      <label
        htmlFor="profile"
      >
        Profile
      </label>
      <div
        className="Select Select--single is-clearable is-searchable"
        style={undefined}
      >
        <div
          className="Select-control"
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          style={undefined}
        >
          <span
            className="Select-multi-value-wrapper"
            id="react-select-2--value"
          >
            <div
              className="Select-placeholder"
            >
              Select...
            </div>
            <div
              className="Select-input"
              style={
                Object {
                  "display": "inline-block",
                }
              }
            >
              <style
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "input#undefined::-ms-clear {display: none;}",
                  }
                }
              />
              <input
                aria-activedescendant="react-select-2--value"
                aria-describedby={undefined}
                aria-expanded="false"
                aria-haspopup="false"
                aria-label={undefined}
                aria-labelledby={undefined}
                aria-owns=""
                className={undefined}
                id={undefined}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                role="combobox"
                style={
                  Object {
                    "boxSizing": "content-box",
                    "width": "5px",
                  }
                }
                tabIndex={undefined}
                value=""
              />
              <div
                style={
                  Object {
                    "height": 0,
                    "left": 0,
                    "overflow": "scroll",
                    "position": "absolute",
                    "top": 0,
                    "visibility": "hidden",
                    "whiteSpace": "pre",
                  }
                }
              >
                
              </div>
            </div>
          </span>
          <span
            className="Select-arrow-zone"
            onMouseDown={[Function]}
          >
            <span
              className="Select-arrow"
              onMouseDown={[Function]}
            />
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col-xs-2"
    id="taxes-state-zipcode"
  >
    <label
      className="boldify zipcode-text"
      htmlFor="Edit Taxes"
    >
      NY
      ,
      11042
    </label>
  </div>
</div>
`;
